doc/tutorial/get_started.rst

```diff
@@ -400,8 +400,8 @@ See the full initialisation process here:
     Using "enp6s0" on "micro4" for OVN uplink
 
    Specify the IPv4 gateway (CIDR) on the uplink network (empty to skip IPv4): 192.0.2.1/24
-   Specify the first IPv4 address in the range to use with LXD: 192.0.2.100
-   Specify the last IPv4 address in the range to use with LXD: 192.0.2.254
+   Specify the first IPv4 address in the range to use for the distributed network(s): 192.0.2.100
```

Comment:
Only if we also change it in the code.

Reply:
The previous question just says "specify the gateway (CIDR) on the uplink network" and this whole block starts with "Do you want to set up distributed networks?" so I think it might be less of a mouthful if we just say

`Specify the {first/last} {ipv4/ipv6} address in the range to use on the uplink network`

Reply:
I actually find Tom's version clearer - yes, it's long and a bit repetitive, but for me who has no clue about networks, this makes it clear that okay, I'm now creating the distributed network, and I need to specify and address on the uplink network.